scripts/generate-icon-exports.js

```diff
@@ -1,49 +1,139 @@
 const fs = require('fs');
+const util = require('util');
 const path = require('path');
 const glob = require('glob');
+const del = require('del');
+const svgr = require('@svgr/core').default;
+
+// Convert fs.readFile / writeFile into Promise version of same
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+const mkdir = util.promisify(fs.mkdir);
```

Comment:
Nit/FYI: you can use fsPromises here I think [ref](https://nodejs.org/api/fs.html#fs_fspromises_access_path_mode).